src/modules/dns.c

```diff
@@ -92,6 +92,17 @@ static void dns_module_dns_ctx_handle_free(void *vh) {
   mtevAssert(h->timeout == NULL);
   free(h);
 }
+static void dns_module_dns_ctx_handle_free_and_remove_eventer(void *vh) {
+  dns_ctx_handle_t *h = vh;
```

Comment:
Should check for NULL on `vh` and return if NULL

Reply:
Fixed